app/lib/features/settings/pages/notifications_page.dart

```diff
@@ -0,0 +1,292 @@
+import 'package:acter/common/notifications/notifications.dart';
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:acter/features/home/providers/client_providers.dart';
+import 'package:acter/features/settings/widgets/settings_section_with_title_actions.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:acter/common/snackbars/custom_msg.dart';
+import 'package:acter/common/utils/utils.dart';
+import 'package:acter/common/widgets/with_sidebar.dart';
+import 'package:acter/features/settings/providers/settings_providers.dart';
+import 'package:acter/features/settings/widgets/settings_menu.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:settings_ui/settings_ui.dart';
+
+class AddEmail extends StatefulWidget {
```

Comment:
Could it belong to ```widgets``` folder ?

Reply:
technically, yes. but if there is no other thing using the widget, I always find it cumbersome to have them in a separate file. It is local to this page, I can give it a `_`-prefixed name, if that matters...